CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.16.0]
```

Comment:
Please make it `[Unreleased]`, as we still don't know what the next release will be.

Reply:
Good catch. I changed the version to `[Unreleased]` :wink: